content/en/tracing/trace_search_and_analytics/_index.md

```diff
@@ -7,9 +7,10 @@ aliases:
 
 {{< vimeo 278748711 >}}
 
-[Trace Search & Analytics][1] is used to filter APM Data by [user-defined tags](#custom-tagging) such as `customer_id`, `error_type` or `app_name` to help troubleshoot and filter your requests. Apply the following configuration to your application to enable this feature.
+[Trace Search & Analytics][1] is used to filter APM Data by user-defined tags such as `customer_id`, `error_type` or `app_name` to help troubleshoot and filter your requests. To enable it either:
 
-To enable it, first configure your services to emit the relevant analytics either [automatically](#automatic-configuration) or [manually](#custom-instrumentation). Then [enable Trace Search inside Datadog][1] in order to start forwarding these analytics.
+* Configure your APM tracer to emit the relevant analytics either [automatically](#automatic-configuration) or [manually](#custom-instrumentation) from your services. Then [enable Trace Search inside Datadog][1] in order to start forwarding these analytics.
```

Comment:
```suggestion
* Configure your APM tracer to emit the relevant analytics from your services—this can be done either [automatically](#automatic-configuration) or [manually](#custom-instrumentation). Next, [enable Trace Search inside Datadog][1] to begin forwarding these analytics.
```